packages/whale-api-client/src/api/loan.scheme.ts

```diff
@@ -0,0 +1,29 @@
+import { WhaleApiClient } from '../whale.api.client'
+import { ApiPagedResponse } from '../whale.api.response'
+import { LoanSchemeResult } from '@defichain/jellyfish-api-core/dist/category/loan'
+
+export class LoanScheme {
+  constructor (private readonly client: WhaleApiClient) {
+  }
+
+  /**
+   * Paginate query loan schemes.
+   *
+   * @param {number} size of scheme to query
+   * @param {string} next set of schemes
+   * @return {Promise<ApiPagedResponse<LoanSchemeResult>>}
+   */
+  async listSchemes (size: number = 30, next?: string): Promise<ApiPagedResponse<LoanSchemeResult>> {
```

Comment:
since the class named `LoanScheme`, just name the CRUD with repeat its class

```suggestion
  async list (size: number = 30, next?: string): Promise<ApiPagedResponse<LoanSchemeResult>> {
```